package.json

```diff
@@ -5,9 +5,8 @@
   "scripts": {
     "build": "tsc -p tsconfig.build.json",
     "start": "node --unhandled-rejections=warn -r ts-node/register/transpile-only -r tsconfig-paths/register -r dotenv/config build/server.js dotenv_config_path=/efs/isomer/.isomer.env",
-    "dev:services": "docker compose up -d",
     "dev:server": "source .env && ts-node-dev --unhandled-rejections=warn --respawn src/server.js",
-    "dev": "npm run dev:services && npm run dev:server",
+    "dev": "docker compose up",
```

Comment:
how about build and start?

Reply:
that's for serving our _production_ site - no need to change

Reply:
ah this is primarily for dev only? ok then

Reply:
yea, on prod we need to write a deploy file to ecr, that will be consuemd to ecs